lib/hypervisor/hv_kvm/monitor.py

```diff
@@ -273,6 +366,30 @@ def __enter__(self):
   def __exit__(self, exc_type, exc_value, tb):
     self.close()
 
+  def execute_qmp(self, command: str, arguments: Dict = None) -> QmpMessage:
+    # During the first calls of Execute, the list of supported commands has not
+    # yet been populated, so we can't use it.
+    if (self.supported_commands is not None and
+        command not in self.supported_commands):
+      raise QmpCommandNotSupported(f"Instance does not support the '{command}'"
+                                    " QMP command.")
+
+    message = QmpMessage({self._EXECUTE_KEY: command})
+    if arguments:
+      message[self._ARGUMENTS_KEY] = arguments
+    logging.debug("QMP JSON Command: %s", message.to_json_string())
+    self.send_qmp(message)
+
+    ret = self.get_qmp_response(command)
+    if command not in [self._QUERY_COMMANDS, self._CAPABILITIES_COMMAND]:
+      logging.debug("QMP Response: %s %s: %s\n", command, arguments, ret)
+    return ret
+
+  # as workaround for existing code
+  # this should be changed in the future!
+  def Execute(self, command, arguments=None):
```

Comment:
I think we should update all calls to `Execute()` to use `execute_qmp()` instead as they are only limited to the hv_kvm implementation and quite easy to spot/replace. I do not see much sense in adding a temporary alias for backwards compatibility.

Reply:
I would leave this as it is for the time being, as I'm still working on another feature that needs to be added first.
Then i would rename this and all other methods according to the desired style